src/main/java/frc/robot/subsystems/Ramp.java

```diff
@@ -74,8 +74,12 @@ public void resetEncoder() {
         neoPidMotor.resetEncoderPosition();
     }
 
-    public double encoderToAngle(double encoderValue){
+    public static double encoderToAngle(double encoderValue){
         //y=mx+b
         return 2.48 * encoderValue + 28.5;//needs be to measured again and put in constants
     }
+    public static double angleToEncoder(double angle){
```

Comment:
why not just add setPosInDegrees()?

Reply:
great question. That makes a lot of sense